services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -172,10 +176,18 @@ async def get_project_for_user(
     """
     db = ProjectDBAPI.get_from_app_context(app)
 
+    product_name = await db.get_project_product(ProjectID(project_uuid))
+    prj_access_rights = await get_user_project_access_rights(
+        app,
+        project_id=ProjectID(project_uuid),
```

Comment:
there are so many copy/paste of that thing. I think you even have a function that does it. Can we have that instead?